src/main/features/ccj/draft/draftCCJ.ts

```diff
@@ -3,19 +3,24 @@ import { CCJPaymentOption } from 'ccj/form/models/ccjPaymentOption'
 import { PaidAmount } from 'ccj/form/models/paidAmount'
 import { PayBySetDate } from 'ccj/form/models/payBySetDate'
 import { RepaymentPlan } from 'ccj/form/models/repaymentPlan'
+import { QualifiedDeclaration } from 'ccj/form/models/qualifiedDeclaration'
 
 export class DraftCCJ {
   defendant: Defendant = new Defendant()
   paymentOption: CCJPaymentOption = new CCJPaymentOption()
   paidAmount?: PaidAmount
   payBySetDate?: PayBySetDate
   repaymentPlan?: RepaymentPlan
+  qualifiedDeclaration?: QualifiedDeclaration
 
-  constructor (defendant: Defendant = new Defendant(), paymentOption: CCJPaymentOption = new CCJPaymentOption(), paidAmount?: PaidAmount, repaymentPlan?: RepaymentPlan) {
+  constructor (defendant: Defendant = new Defendant(), paymentOption: CCJPaymentOption = new CCJPaymentOption(),
```

Comment:
I think it's better to have each one on a single line in case it gets too long.

Reply:
done